README.md

```diff
@@ -1,16 +1,229 @@
+* **Repetition**: Given a need to something until a something is `true`, or
+  until something has happened a certain number of times, can we change the
+  sequence to make something repeat before moving on?
+
+Let's look at a few examples. Try them out in IRB. We'll cover them more
+formally later.
+
+> **NOTE**: These examples assume you're working in a
+fresh, new IRB session _for each example_. If you define `result` and _don't_
+restart IRB and a later example might assume `result` is `nil`, you might get a
+surprising result. Use `exit` to leave IRB and launch a fresh session if you're
+surprised by a result.
+
+## Provide Example of Sequence Statement
+
+The default **sequence** of code is: "top to bottom, every line, left to right
+as ruled by order of operations"
+
+```ruby
+result = 1 + 1
+puts result #=> 2
+```
+
+You probably have an intuitive belief that the default sequence is this because
+you recognize code as "text" and you probably have the general theory that
+English text is read "top to bottom, left to right."
+
+```ruby
+puts result #=> Error
+result = 1 + 1
+```
+
+This error makes sense, because you're trying to do variable lookup before
+setting the variable.
+
+## Provide Example of Selection Statement
+
+Sometimes we need to deviate from the default **sequence** we might need to
+**select** a different path. There's a poem by Robert Frost about it:
+
+> Two roads diverged in a yellow wood,
+> And sorry I could not travel both
+> And be one traveler
+
+In this case the traveler is Ruby, traveling fatefully down the default
+sequence. We, as programmers, create a divergence, a "split" in fate, and ask
+Ruby to take one path (or the other, or a third, or a fourth...and so on) based
+on a Boolean "test" _evaluation_. We ask Ruby to _select_ the path.
+
+The first tool we'll learn to do **selection** is `if`. It disrupts the
+"default sequence" by asking Ruby to run a test, decide whether to follow the
+path, and then move back to the default sequence.
+
+```ruby
+favorite_number = 2 * 2 * 2 * 2 * 2 * 2 * 2 * 2
+if favorite_number >= 10
+  favorite_number = favorite_number + 10
+  puts "And now your favorite_number is 10 more!"
+end
+puts "THE END"
+```
+
+**Selection** lets us disrupt default flow by _making a choice_.
+
+## Provide Example of Repetition Statement
+
+**Repetition** lets us disrupt default flow by _repeating_. The `.times`
+method, which will introduce formally in just a moment, means "do something
+<value> times." That "something" is held inside a `do...end` block. Other
+programming languages like JavaScript like to use (`{}`) to set up a "block of
+stuff to do". Python likes to use an indented line of code and neither `{}` nor
+`do...end`. Ruby prefers `do...end`.
+
+```ruby
+favorite_number = 2 * 2 * 2 * 2 * 2 * 2 * 2 * 2
+10.times do
+  favorite_number = favorite_number + 1
+end
+puts "And now your favorite_number is 10 more! It is #{favorite_number}"
+```
 
 ## Conclusion
 
-## Resources
+(wip)
```

Comment:
In a writing class, there is value in establishing a common vocabulary for writing. Once we understand what sentences are, we can talk about correct ways to structure them. As we get more comfortable writing sentences, we can get more creative. Rearrange the clauses, chain sentences together, create narrative.

Before we can write eloquent code, we need to be able to recognize the _sentences_ and _paragraphs_ of programming. Understanding what sequence, selection and repetition statements are allows us to get creative with them, chain them together, compose longer and longer conversations with Ruby.